playground/html/nested/index.html

```diff
@@ -1,3 +1,3 @@
-<link rel="stylesheet" href="./nested.css" />
+<link rel="stylesheet" href="nested.css" />
 <h1>Nested</h1>
-<script type="module" src="./nested.js"></script>
+<script type="module" src="nested.js"></script>
```

Comment:
Maybe it is better to leave `./` in either of them?

Reply:
Better to have both?